viz/metrics-api/stat_summary.go

```diff
@@ -77,8 +77,15 @@ func (s *grpcServer) StatSummary(ctx context.Context, req *pb.StatSummaryRequest
 	}
 
 	// err if --from is added with policy resources
-	if req.GetFromResource() != nil && isPolicyResource(req.GetSelector().GetResource()) {
-		return statSummaryError(req, "'from' queries are not supported with policy resources, as they have inbound metrics only"), nil
+	if req.GetFromResource() != nil {
+		if isPolicyResource(req.GetSelector().GetResource()) ||
+			isPolicyResource(req.GetFromResource()) {
+			return statSummaryError(req, "'from' queries are not supported with policy resources, as they have inbound metrics only"), nil
+		}
+	}
+
+	if req.GetToResource() != nil && isPolicyResource(req.GetToResource()){
```

Comment:
I think we're missing the case here there is a toResource and the selector is a policy resource.  That should return an error too.

To put it another way: if either `--to` or `--from` are specified, none of the involved resources can be policy resources.